tests/trainer/test_trainer_tricks.py

```diff
@@ -112,33 +112,46 @@ def test_overfit_batch_limits(tmpdir):
     # ------------------------------------------------------
     for split in (RunningStage.VALIDATING, RunningStage.TESTING):
 
+        # ------------------------------------------------------
+        # test overfit_batches action
+        # ------------------------------------------------------
+
```

Comment:
Why was this added?

Reply:
The test cases below are all sub-test of it. Or I can remove it.